float32 angle
uint64 x_pos
int16 y_pos
string direction
uint8 lane_detect
